adaptive_navigation/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
 # [0.0.1] - August 12, 2020
 
 The initial release adds [adaptive navigation](https://material.io/design/layout/responsive-layout-grid.html#ui-regions)
+
+# [0.0.2] - October 9, 2020
```

Comment:
I think changelogs typically go from most recent at the top to least recent at the bottom

Reply:
Done